vita3k/gui/src/settings_dialog.cpp

```diff
@@ -218,34 +208,37 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             if (host.cfg.pref_path.back() != '/')
                 host.cfg.pref_path += '/';
             if (host.cfg.pref_path != host.pref_path) {
-                if (change_pref_location(host.cfg.pref_path, host.pref_path)) {
+                //if (change_pref_location(host.cfg.pref_path, host.pref_path)) {
                     host.pref_path = host.cfg.pref_path;
 
                     // Refresh the working paths
                     config::serialize_config(host.cfg, host.cfg.config_path);
-                    LOG_INFO_IF(clear_and_refresh_game_list(host, gui), "Successfully moved Vita3K files to: {}", host.pref_path);
-                }
+                    refresh_game_list(gui, host);
+                    LOG_INFO("Successfully change Vita3K path files to: {}", host.pref_path);
+                //}
             }
         }
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("After pressing, restart Vita3K to fully apply changes.");
+            ImGui::SetTooltip("Change Vita3K emulator path like wanted.\nTODO: Need move folder old to new manualy.");
```

Comment:
The `TODO:` in user text?

Reply:
hmm yes, for said just for moment is not implemented

Reply:
Would be good as a comment. Not sure if the user wants to see it.

Reply:
i have set comment for that ^^

Reply:
I don't see it... 🤔